serpent-moltres/isotope-distribution/compact-hete

```diff
@@ -97,8 +97,22 @@ set acelib "/projects/sciteam/bbcc/serpent/xsdata/jeff312/sss_jeff312.xsdata"
 set bc 3
```

Comment:
Add that this is the heterogenous model in the description at top of file

Reply:
done